acceptance/bundle/telemetry/deploy/databricks.yml

```diff
@@ -0,0 +1,18 @@
+bundle:
+  name: rabbit
+  uuid: 11111111-2222-3333-4444-555555555555
+
+resources:
```

Comment:
Testing other resources requires a rich fake workspace abstraction. I'll try to see if I can autogenerate that bit as a followup and thus make the testing here more complete.